ipaserver/install/ntpinstance.py

```diff
@@ -88,37 +71,19 @@ def __write_config(self):
                 ntpconf.append(line)
                 continue
 
-            if opt[0] == "server":
-                match = False
-                for srv in srv_vals:
-                    if opt[1] == srv:
-                        match = True
-                        break
-                if match:
-                    srv_vals.remove(srv)
-                else:
-                    file_changed = True
-                    line = ""
+            if opt[0] == "server" and opt[1] == local_srv:
```

Comment:
This will blow up if a config line contains only single token. This an easily happen if your config contains some comments, e.g.
~~~
#####################################
# BEWARE: Do not touch this config! #
#####################################
~~~